server/athenian/api/openapi/openapi.yaml

```diff
@@ -838,6 +838,219 @@ paths:
 # {% endif %}
       x-openapi-router-controller: athenian.api.controllers.reposet_controller
       x-codegen-request-body-name: body
+  /team/create:
+    post:
+      operationId: create_team
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TeamCreateRequest'
+        x-body-name: body
+      responses:
```

Comment:
How about 422 Unprocessable Entity. We raise 400 on JSON syntax errors, so if we can find a more specific code for this error then it will be distinguishable.

Reply:
`422` lgtm

Reply:
good point, I'd just go for 400 with different error message. I think that distinguishing these two different cases by different status code will be important when and if we're going to have clients different from the webapp.

Reply:
done [here](https://github.com/athenianco/athenian-api/pull/334/commits/6ce2316202b0863ecbb1a3a98385b49771ffc446).